packages/pipeline-editor/style/index.css

```diff
@@ -81,12 +81,12 @@
 }
 
 .lm-Widget #jp-left-stack {
-  padding-left: 8px;
+  padding-left: unset;
 }
 
 .lm-Widget #jp-right-stack {
-  margin-top: 25px;
-  margin-right: 5px;
+  margin-top: 0px;
+  margin-right: 0px;
```

Comment:
if this is unnecessary we should just remove it instead

Reply:
I agree. Still looking at how the deletion can be picked up, as I tried a clean build and clearing browser cache as suggested but it didn't help

Reply:
I traced back to the original commit https://github.com/elyra-ai/elyra/commit/b842f49ae095df4ef98b11f88d07e1a52356b259 and I'm not sure these ever were needed

Reply:
I just realized after noting that the pr number for that commit didn't add up that the commit is from when the button was still its own repo. Back then it had it's own ui that opened in the right sidebar. We must have not removed the css when we removed that part of the ui

Reply:
I tested with these deleted and it looks great.